lib/simple_token_authentication.ex

```diff
@@ -12,19 +12,19 @@ defmodule SimpleTokenAuthentication do
 
   def call(conn, _opts) do
     val = get_auth_header(conn)
-    
-    tokens =
-      :simple_token_authentication
-      |> Application.get_env(:token)
-      |> List.wrap()
-      
-    service_tokens =
-      :simple_token_authentication
-      |> Application.get_env(:service_tokens)
-      |> List.wrap()
 
-    if Enum.any?(tokens ++ service_tokens, &matches?(&1, val)) do
-      conn
+    token_map =
+      case :persistent_term.get(:simple_token_authentication_map, nil) do
```

Comment:
I don't know a lot about persistent_term. It looks like it's more read-optimized than ets. But would an ets table with a row for each token be clearer than a map in persistent_term?

Reply:
persistent term are fastest to read compare to ETS, as long as you keep low writing which should be the case in here since we should configure once and keep the term forever that way?